support-frontend/app/controllers/CreateSubscriptionController.scala

```diff
@@ -150,15 +160,30 @@ class CreateSubscriptionController(
         }
     }
 
-  private def logIncomingRequest(
-      request: OptionalAuthRequest[CreateSupportWorkersRequest],
-      maybeLoggedInIdentityIdAndEmail: Option[IdentityIdAndEmail],
-  ) = {
+  private def errorDetailedMessage(message: String)(implicit request: CreateRequest) = {
+    val maybeLoggedInIdentityIdAndEmail =
+      request.user.map(authIdUser => IdentityIdAndEmail(authIdUser.id, authIdUser.primaryEmailAddress))
+
     val userDesc = maybeLoggedInIdentityIdAndEmail match {
-      case None => s"Guest User ${request.body.email}"
-      case Some(idAndEmail) => s"User ${idAndEmail.primaryEmailAddress}"
+      case None => (request.body.email, "guest")
+      case Some(idAndEmail) => (idAndEmail.primaryEmailAddress, "logged-in")
     }
-    SafeLogger.info(s"$userDesc is attempting to create a new ${request.body.product.describe} [${request.uuid}]")
+    SafeLogger.error(
+      scrub"[CreateSubscriptionController] [${request.uuid}] [${userDesc._1}] [${userDesc._2}] [${request.body.product.describe}] $message",
+    )
+  }
+
+  private def logDetailedMessage(message: String)(implicit request: CreateRequest) = {
+    val maybeLoggedInIdentityIdAndEmail =
+      request.user.map(authIdUser => IdentityIdAndEmail(authIdUser.id, authIdUser.primaryEmailAddress))
+
+    val userDesc = maybeLoggedInIdentityIdAndEmail match {
+      case None => (request.body.email, "guest")
+      case Some(idAndEmail) => (idAndEmail.primaryEmailAddress, "logged-in")
+    }
+    SafeLogger.info(
+      s"[CreateSubscriptionController] [${request.uuid}] [${userDesc._1}] [${userDesc._2}] [${request.body.product.describe}] $message",
+    )
```

Comment:
I usually avoid _1 and _2
```suggestion
    val (user, desc) = maybeLoggedInIdentityIdAndEmail match {
      case None => (request.body.email, "guest")
      case Some(idAndEmail) => (idAndEmail.primaryEmailAddress, "logged-in")
    }
    SafeLogger.info(
      s"[CreateSubscriptionController] [${request.uuid}] [$user] [$desc] [${request.body.product.describe}] $message",
    )
```